src/python/grpcio_tests/tests/interop/BUILD.bazel

```diff
@@ -88,6 +88,18 @@ py_library(
     ],
 )
 
+py_binary(
```

Comment:
is this leftover code?

Reply:
(from debugging)

Reply:
It would be nice to add this `py_binary` rule. We have a similar one for the interop client.

Reply:
cool!